### PR TITLE
Invite page and rsvp styling adjustments

### PR DESCRIPTION
**Pull request by kyle-sablan**
- Invite page similar in design to wireframe
- added in registry on invite page
- made deadline only viewable in rsvp form page
- incorporated common styles

### CODE REVIEW

File: src/GuestPages/Invite.js [hunk 1]
```diff
@@ -18,14 +23,47 @@ const Invite = () => (
...
+          After our 12-week coding bootcamp, you'll be what you want to be: a
+          software engineer, fully capable of tackling unique and unfamiliar
+          problems and building complex applications on the job. To help you
+          land that job, we train you to have in-demand technical skills, clear
+          communication skills, and the ability to work autonomously. You'll
+          learn JavaScript, you'll learn to code, and so much more. Watch this
+          short video to learn how. We are currently offering our 12-week
+          bootcamp in a live online format due to current conditions. Onsite
+          offerings are not available at this time.
+        </BMerr18>
```
Comment (hithisiskhai) [inline src/GuestPages/Invite.js:52]: What a strange couple.

**Review by hithisiskhai — APPROVED** [review]
Sizing looks better. Good use of common components where applicable.

File: src/GuestPages/InfoRSVP.js [hunk 2]
```diff
@@ -1,75 +1,61 @@
...
-        </TempHeader>
+        </BAlexBrushH1>
       </>
     )}
-    <TempHeader
+    <BFaustinaH2
       className="d-flex justify-content-center"
       style={{
-        fontSize: '28px',
+        fontStyle: 'italic',
```
Comment (DylanDKim) [inline src/GuestPages/InfoRSVP.js:31]: curious to see what it looks like in italic

File: src/GuestPages/Invite.js (hunk 1 above)
Comment (DylanDKim) [reply src/GuestPages/Invite.js:52]: lmao

**Review by DylanDKim — APPROVED** [review]

**Pull request merged**